products/mssqlserver.md

```diff
@@ -1,55 +1,67 @@
 ---
 title: Microsoft SQL Server
-permalink: /mssqlserver
-iconSlug: microsoftsqlserver
 category: db
+iconSlug: microsoftsqlserver
+permalink: /mssqlserver
+versionCommand: select @@version
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
+
 activeSupportColumn: true
-versionCommand: select @@version
+releaseColumn: true
 releaseDateColumn: true
+extendedSupportColumn: Extended Security Updates
+
 releases:
```

Comment:
When `extendedSupportColumn` is active, the `extendedSupport` must be specified for all release cycles (same constraint as `activeSupportColumn`).

Reply:
Fixed.